ios/FluentUI.Demo/FluentUI.Demo/Demos/SearchBarDemoController.swift

```diff
@@ -33,7 +51,10 @@ class SearchBarDemoController: DemoController, SearchBarDelegate {
     }
 
     func searchBar(_ searchBar: SearchBar, didUpdateSearchText newSearchText: String?) {
-        searchBar.placeholderText = newSearchText ?? "search"
+        if searchBar == searchBarWithBadgeView && searchBar.searchText?.lowercased() == "badge" && searchBarWithBadgeView.badgeView == nil {
```

Comment:
We should be able to demonstrate how the search bar would look with multiple badges as well, along with custom input badges. Thoughts?

Reply:
That's a good consideration. The way it is implemented today it seems like there is only space for a single leading view in the Search Bar. This seems to be by design and it does not attempt to reproduce the behavior of the BadgeField.

@zhouyuzhe88 could you confirm the assumption?

Reply:
It's by design that we only need one badge. More info is added into related issue #870.
Supporting multiple badges seems to be an over engineering: usesr may need to search files from A, but it's rarely that a user want's to search files from A and B.
Another question for multiple badges is that will the badges means AND or OR?
Let's support only one badge in the search bar.